.changeset/wet-roses-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@threlte/extras': patch
```

Comment:
this is a `minor` update

Reply:
That's what I wanted to do, is it safe to just edit the changeset .md file like I did?

I think I chose a minor, then patch and it got overwritten..